### PR TITLE
Product model and controller

### PR DESCRIPTION
**Pull request by Rinbo**
## Product Model and Controller

PT: https://www.pivotaltracker.com/story/show/160725648

- Create model Product for products with columns 'name', 'descriptions' and 'price'
- Create a controller for Product model with corresponding action and views
- Rspec unit tests for checking for the correct columns and validations

### CODE REVIEW

**Comment by magnus-thor** [comment]
Cant view the pivotal tracker story.
This PR is missing a feature file

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class ProductsController < ApplicationController
+  def index
+    @products = Product.all
+  end
+
+  def show
```
Comment (magnus-thor) [inline app/controllers/products_controller.rb]: The show, new and create methods are out of scope.

**Review by magnus-thor — DISMISSED** [review]

File: app/controllers/products_controller.rb (hunk 1 above)
Comment (magnus-thor) [reply app/controllers/products_controller.rb]: And the PT link is still not working

**Review by tochman — APPROVED** [review]
Good job

**Pull request merged**